integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXScaleTest.java

```diff
@@ -1,72 +1,54 @@
 package com.sequenceiq.it.cloudbreak.testcase.e2e.distrox;
 
+import java.util.stream.IntStream;
+
 import javax.inject.Inject;
 
 import org.testng.ITestContext;
 import org.testng.annotations.Test;
 
-import com.sequenceiq.distrox.api.v1.distrox.model.database.DistroXDatabaseAvailabilityType;
-import com.sequenceiq.it.TestParameter;
 import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
 import com.sequenceiq.it.cloudbreak.context.Description;
 import com.sequenceiq.it.cloudbreak.context.TestContext;
 import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
-import com.sequenceiq.it.cloudbreak.dto.distrox.externaldatabase.DistroXExternalDatabaseTestDto;
-import com.sequenceiq.it.cloudbreak.dto.distrox.image.DistroXImageTestDto;
 import com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest;
 
 public class DistroXScaleTest extends AbstractE2ETest {
-    private static final String DIX_EXTDB_KEY = "distroxExternalDatabaseKey";
 
     @Inject
     private DistroXTestClient distroXTestClient;
 
-    @Inject
-    private TestParameter testParameter;
-
     @Override
     protected void setupTest(TestContext testContext) {
         testContext.getCloudProvider().getCloudFunctionality().cloudStorageInitialize();
         createDefaultUser(testContext);
         createDefaultCredential(testContext);
         initializeDefaultBlueprints(testContext);
-        createEnvironmentWithFreeIpa(testContext);
-        createDatalake(testContext);
+        initiateEnvironmentCreation(testContext);
+        initiateDatalakeCreation(testContext);
+        waitForEnvironmentCreation(testContext);
+        waitForUserSync(testContext);
+        waitForDatalakeCreation(testContext);
```

Comment:
It would be better in AbstractE2ETest as a standalone method. Maybe it could be the default process to create the env and sdx together (I think it is much quicker with external db)

Reply:
yes, that would be the plan, but first I would enable it only for this test first, to not break anything in worse case